_beats/libbeat/_meta/kibana/default/index-pattern/libbeat.json

```diff
@@ -12,5 +12,5 @@
       "version": 1
     }
   ],
-  "version": "6.1.0"
+  "version": "6.1.1"
```

Comment:
Not related to this PR but I think this file should not be checked in in apm-server. Need to double check.